src/test/java/org/opensearch/security/dlic/rest/api/UserApiTest.java

```diff
@@ -1004,4 +1010,26 @@ public void checkNullElementsInArray() throws Exception {
         Assert.assertEquals(RequestContentValidator.ValidationError.NULL_ARRAY_ELEMENT.message(), settings.get("reason"));
     }
 
+    @Test
+    public void testGeneratedPasswordContents() {
+        String password = UserService.generatePassword();
```

Comment:
Nit: Add a test that makes sure two calls produce different results

Reply:
Sure will do